website/tutorial/basic/06-adding-a-database.mdx

```diff
@@ -0,0 +1,251 @@
+---
+id: 06-adding-a-database
+title: Adding a Database
+sidebar_label: Adding a Database
+---
+
+In this section, you're going to set up a [SQLite](https://www.sqlite.org/index.html) to persist the data of incoming GraphQL mutations. Instead of writing SQL directly, you will
```

Comment:
I would maybe add here that GraphQL can connect to many types of sources like DB or remote APIs.
In this chapter we are covering a DB but same work could be replaced by calling a remote API